docs/TryItOut.md

```diff
@@ -8,7 +8,7 @@ You can also follow along in this video:
 
 ## Create your first invoice
 
-For an in production setup, skip to [Mainnet](TryItOut.md#Mainnet)
+For an in production setup, skip to [Mainnet](TryItOut.md#mainnet)
```

Comment:
This seems outdated as it was already fixed with 630fabc110e34e89fbae59dea55633dc5f885a8e

Reply:
```suggestion
For an in production setup, skip to [Mainnet](TryItOut.md#Mainnet)
```

Reply:
The lowercase version was correct – I meant to point out that this has been fixed in masdter already. Sorry for the misunderstanding, my bad!